scenario_player/runner.py

```diff
@@ -371,7 +359,7 @@ def _initialize_udc(
         return ud_token_tx, udc_ctr, should_deposit_ud_token
 
     def _initialize_nodes(
-        self,
+        self, nursery: Any
```

Comment:
Can't we type this properly?

Reply:
The `Nursery` is defined inside the `Janitor` as a closure. I don't think mypy can access that. We can probably pull that out into a separate class with some additional constructor args. But that has to be done on the Janitor side first.

Reply:
There's `class Nursery(ABC):` for that reason I think.

Reply:
Good point, I forgot about that!